pipeline.yml

```diff
@@ -725,13 +757,21 @@ jobs:
       params: *bosh-params-tooling
   - put: semver-tooling-version
     params: {bump: patch}
+  - task: update-cloud-config
```

Comment:
WDYT about moving this task into the deploy-bosh job?  To me it seems like it belongs there more than with the release uploads.